packages/web/src/pages/track-page/components/mobile/TrackHeader.tsx

```diff
@@ -224,11 +227,15 @@ const TrackHeader = ({
     { enabled: !!trackId }
   )
 
+  // @ts-ignore - artwork exists on track, but was previously removed so it's not on the type
+  const staticImage = track?.artwork?.[SquareSizes.SIZE_480_BY_480]
```

Comment:
so i dont think artwork is what we want to do for ssr? wouldnt it be "track.cover_art_sizes" and from there we get the url?

Reply:
getImageUrl in the landing page is a good example of what we might consider for ssr flow.